packages/css/src/List/List.stories.ts

```diff
@@ -9,10 +9,15 @@ const meta: Meta = {
 export default meta;
 
 export const Default: StoryObj = {
-  render: (args) => {
+  render: ({ classModifier, ...args }) => {
     const list = document.createElement("section");
 
     list.className = "af-card af-card--list";
+
+    if (classModifier) {
+      list.className += ` af-card--${classModifier}`;
+    }
```

Comment:
classModifier ressemble par exemple à ça : "primary disabled".
Pour rajouter les classNames suivantes : "af-component--primary af-component--disabled".
Du coup ` af-card--${classModifier}` ne fonctionnera pas dans le cas où il y a deux classModifiers.
Voici un code qui fonctionne : 

```suggestion
    list.className = `af-card${args.classModifier
      .split(" ")
      .filter(Boolean)
      .map((classModifier: string) => ` af-card--${classModifier}`)
      .join("")}`;
```